src/scrabble.js

```diff
@@ -1,6 +1,55 @@
+const letters = {
+  value1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T" ,'a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't'],
+  value2: ["D", "G", "d", "g"],
+  value3: ["B", "C", "M", "P", "b" ,"c" ,"m", "p"],
+  value4: ["F", "H", "V", "W", "Y", "f", "h", "v", "w", "y"],
+  value5: ["K", "k"],
+  value8: ["J", "X" ,"j" ,"x"],
+  value10: ["Q", "Z", "q", "z"]
+}
```

Comment:
instead of typing in small and capital letters each, I used .toLowerCase() on the letter in the function.